plugins/dominant-color-images/load.php

```diff
@@ -5,7 +5,7 @@
  * Description: Displays placeholders based on an image's dominant color while the image is loading.
  * Requires at least: 6.6
  * Requires PHP: 7.2
- * Version: 1.1.2
+ * Version: 1.1.3
```

Comment:
I suggest perhaps reverting the version changes since we typically bump all those when doing a release. For example, at the time of release we can evaluate what type of version bump is warranted, whether a patch or a minor version bump.

```suggestion
 * Version: 1.1.2
```